src/ui/control-panel.ts

```diff
@@ -66,6 +67,216 @@ class ControlPanel extends Panel {
         splatSize.append(splatSizeLabel);
         splatSize.append(splatSizeSlider);
 
+        // highlighting color of selected splats (splat and ring)
+        const selectedSplatColor = new Container({
+            class: 'control-parent'
+        });
+
+        const selectedSplatColorLabel = new Label({
+            class: 'control-label',
+            text: 'Highlight Splat Color'
+        });
+
+        const selectedSplatColorPicker = new ColorPicker({
+            class: 'control-element-expand',
+            value: [1.0,1.0,0.0]
+        });
+
+        selectedSplatColor.append(selectedSplatColorLabel);
+        selectedSplatColor.append(selectedSplatColorPicker);
+
+        // toggle splats
+        const splatDisplayToggle = new Container({
+            class: 'control-parent'
+        });
+
+        const splatDisplayToggleLabel = new Label({
+            class: 'control-label',
+            text: 'Show Splats'
+        });
+
+        const splatDisplayToggleCb = new BooleanInput({
+            class: 'control-element',
+            value: true
+        });
+
+        splatDisplayToggle.append(splatDisplayToggleLabel);
+        splatDisplayToggle.append(splatDisplayToggleCb);
+
+
+        // color of center points
+        const centerPointColor = new Container({
+            class: 'control-parent'
+        });
+
+        const centerPointColorLabel = new Label({
+            class: 'control-label',
+            text: 'Center Point Color'
+        });
+
+        const centerPointColorPicker = new ColorPicker({
+            class: 'control-element-expand',
+            value: [0.0,0.0,1.0]
+        });
+
+        centerPointColor.append(centerPointColorLabel);
+        centerPointColor.append(centerPointColorPicker);
+
+        // alpha of center points
+        const centerPointAlpha = new Container({
+            class: 'control-parent'
+        });
+
+        const centerPointAlphaLabel = new Label({
+            class: 'control-label',
+            text: 'Center Point Alpha'
+        });
+
+        const centerPointAlphaSlider = new SliderInput({
+            class: 'control-element-expand',
+            precision: 2,
+            min: 0,
+            max: 0.5,
+            value: 0.5
+        });
```

Comment:
Why have a separate slider for alpha here? You can pass `components: 4` to the `ColorPicker` constructor to add support for an alpha component.

Reply:
i see, didnt know. i will implement it like that :)

Reply:
i replaced both alpha sliders and the interpolation slider with alpha channels in color pickers